types/math/dec.go

```diff
@@ -146,11 +146,18 @@ func (x Dec) String() string {
 	return x.dec.Text('f')
 }
 
+// Cmp compares x and y and returns:
+//
+//   -1 if x <  y
+//    0 if x == y
+//   +1 if x >  y
+//   undefined if d or x are NaN
+//
 func (x Dec) Cmp(y Dec) int {
 	return x.dec.Cmp(&y.dec)
 }
 
-func (x Dec) IsEqual(y Dec) bool {
+func (x Dec) Equal(y Dec) bool {
```

Comment:
What's the logic behind this change? Should we keep this as `IsEqual` so that it's consistent with the other methods (i.e. `IsZero`, `IsNegative`, `IsPositive`)?